cylc/flow/cfgspec/workflow.py

```diff
@@ -785,9 +785,21 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     :ref:`SequentialTasks`.
             ''')
 
+        Conf('sequential xtriggers default', VDR.V_BOOLEAN, False, desc='''
+            Set to ``True``, this allows for sequential spawning of associated
+            parentless tasks on xtrigger satisfaction.
+            Instead of out to the runahead limit (default: ``False``).
+
+            This workflow wide default can be overridden by a reserved
+            keyword argument in the xtrigger function declaration
+            (``sequential=True/False``).
+
+            The presence of one sequential xtrigger on a parentless task with
+            multiple xtriggers will cause sequential behavior.
+        ''')
```

Comment:
I reserve and strip `sequential` out of the args when it's read in...
I could undo the reservation and read in the function args (as I did for the reservation), but it would have to be well known, so people don't use `sequential` arg name for some other purpose (I could definitely change the arg type).

And I disagree, on not being able to set a workflow wide default.. Again some sites have hundreds/thousands of `wall_clock` and `workflow_state`.. And I would need to write in the argument to nearly all of them if we have no ability to change the default.. I would rather just

I don't want a global default (in the sense of a user/site setting), because this would be bad for sharing a workflow (as workflows may act differently for different users).

Reply:
Done, it's now an option to add `sequential=True/False` to the function (as well as declaration) .. It will error if is does not have a default or default is not `bool`:
![image](https://github.com/cylc/cylc-flow/assets/11400777/3acdfd4b-03c0-4aa1-bf0a-891206e54367)